solaris/tile/vector_tile.py

```diff
@@ -103,6 +103,7 @@ def tile(self, src, tile_bounds, tile_bounds_crs=None, geom_type='Polygon',
                                                        output_ext))
             self.tile_paths.append(dest_path)
             if len(tile_gdf) > 0:
+                assert tile_gdf.crs is not None
```

Comment:
Thank you for adding this. Can we add an expression for what the error will tell users? e.g.,

```python
if len(tile_gdf) > 0:
    assert tile_gdf.crs is not None, "No CRS was found. A CRS must be present to tile the object."
```

Reply:
Why not raise an exception?

Reply:
I was debating that approach too. I guess I see the exception as probably being more transparent, so that might be a better approach @rbavery.